app/controllers/organisations_controller.rb

```diff
@@ -57,6 +57,15 @@ def show
 
 private
 
+  def policies
+    rummager_results = Whitehall.unified_search_client.unified_search(
+      filter_organisations: [@organisation.slug],
+      filter_format: "policy"
```

Comment:
I suggest adding `count: 3,` to the parameters - this will avoid unneccesary data transfer from rummager to whitehall.

You could also add a trailing comma to the last parameter, I think?